src/bundle/Resources/views/admin/search/search_form.html.twig

```diff
@@ -55,6 +55,29 @@
         </svg>
         <ul class="ez-filters__user-list ez-filters__user-list--hidden"></ul>
     </div>
+    <div class="ez-filters__item ez-filters__item--subtree">
+        <label class="ez-filters__item-label">{{ 'search.subtree'|trans|desc('Subtree:') }}</label>
+        {% set subtree = form.vars.data.subtree %}
+        <button data-universal-discovery-title="{{'search.udw.select_content'|trans|desc('Select Content')}}"
+                class="ez-button-tree pure-button ez-font-icon ez-button btn btn-secondary ez-btn--udw-select-location"
```

Comment:
Too big indentation. Please, stick to 4 spaces.

Reply:
Another thing, I believe there are too many different classes for buttons. I believe they override each other. What's your take on this @dew326 ?

Reply:
Well, I think the `pure-button` is from PlatformUI v1.